packages/cli/src/bin.ts

```diff
@@ -63,15 +63,6 @@ process.once('SIGINT', () => {
   process.exit(130)
 })
 
-if (process.argv.length > 1 && process.argv[1].endsWith('prisma2')) {
-  console.log(
-    yellow('deprecated') +
-      `  The ${red('prisma2')} command is deprecated and has been renamed to ${green('prisma')}.\nPlease execute ${bold(
-        green('prisma' + (commandArray.length ? ' ' + commandArray.join(' ') : '')),
-      )} instead.\n`,
-  )
-}
```

Comment:
This removes the deprecation warning code.

Reply:
I removed the green and yellow imports, which I found in the linting errors as unused